helm-chart/values.yaml

```diff
@@ -6,12 +6,14 @@ tap:
     imagePullSecrets: []
   proxy:
     worker:
-      srvPort: 8897
+      port: 30000
+      srvPort: 30000
     hub:
-      port: 8898
-      srvPort: 8898
+      port: 30001
+      srvPort: 30001
     front:
-      port: 8899
+      port: 30002
```

Comment:
Don't change the defaults.

Reply:
@kcns008 did you address @mertyildiran's change request?